modules/acs-central-services-overview.adoc

```diff
@@ -6,15 +6,17 @@
 = Central services
 
 You install Central services on a single cluster.
-These services include three main components, Central, Central DB and Scanner.
+These services include four main components, Central, Central DB, StackRox Scanner, and Scanner V4 (optional).
```

Comment:
curious, if we specify Central DB as a component, then should we specify Scanner DB and Scanner V4 DB as well as split Scanner V4 into the two parts: Scanner V4 Indexer and Scanner V4 Matcher? That would make it 7 components

Reply:
Might as well? It would be clearer to the reader.

Reply:
I need to go back and add descriptions for these.